### PR TITLE
Fix: Support JVM version string from old (< v8.9) and new (>= v8.9) gradle version

### PR DESCRIPTION
**Pull request by jeffryang24**
*Issue #, if available:* This PR should resolve https://github.com/aws/aws-lambda-builders/issues/682

*Description of changes:*

- Check both `JVM` and `Launcher JVM` prefix for the `gradle[w] -version` command output. Gradle < v8.9 was using `JVM: x.x.x`, while Gradle >= v8.9 has changed it to `Launcher JVM: x.x.x`.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: aws_lambda_builders/workflows/java_gradle/gradle_validator.py [hunk 1]
```diff
@@ -83,5 +83,5 @@ def _get_jvm_string(self, gradle_path):
 
         for line in stdout.splitlines():
             l_dec = decode(line)
-            if l_dec.startswith("JVM"):
+            if l_dec.startswith("JVM") or l_dec.startswith("Launcher JVM"):
```
Comment (hnnasit) [inline aws_lambda_builders/workflows/java_gradle/gradle_validator.py:86]: nit: This could be simplified to `l_dec.contains("JVM")`

**Review by hnnasit — APPROVED** [review]

File: aws_lambda_builders/workflows/java_gradle/gradle_validator.py (hunk 1 above)
Comment (jeffryang24) [reply aws_lambda_builders/workflows/java_gradle/gradle_validator.py:86]: I see, I will change it with `.contains(...)` 👍

Comment (jeffryang24) [reply aws_lambda_builders/workflows/java_gradle/gradle_validator.py]: Updated with `in` operator to check the JVM substring.

Comment (jeffryang24) [reply aws_lambda_builders/workflows/java_gradle/gradle_validator.py]: Hi @hnnasit , is there something that I need to do so this PR can be merged? Thank you 🙇

Comment (lucashuy) [reply aws_lambda_builders/workflows/java_gradle/gradle_validator.py]: Hey @jeffryang24, our CI checks are failing consistently (and randomly out of the blue). The failures are unrelated to your changes, but we're working on fixing them. At the moment, there is no action needed from your side. Thanks for waiting

Comment (jeffryang24) [reply aws_lambda_builders/workflows/java_gradle/gradle_validator.py]: I see, noted 👍

**Review by mndeveci — APPROVED** [review]

**Pull request merged**